build.gradle.kts

```diff
@@ -26,41 +34,42 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
   implementation("org.springframework.boot:spring-boot-starter-mail")
-  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.3.0")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:2.1.1")
```

Comment:
Not sure this is being used at all? If not then might be better to drop it

Reply:
removed - hmpps-sqs-spring-boot-starter dependency.